etna/packages/etna-js/upload/workers/uploader.jsx

```diff
@@ -447,7 +457,8 @@ export class Uploader {
         blob_data,
         next_blob_size: newBlobSize,
         next_blob_hash: newBlobHash,
-        current_byte_position
+        current_byte_position,
+        metis_uid
```

Comment:
Because we're starting the upload with a Magma HMAC-signed URL, we need to provide the Metis UID (since we're not uploading from Metis directly). This has to be passed in from Timur.